src/main/resources/theme/vault/replay/replay_card.fxml

```diff
@@ -84,11 +84,24 @@
             <Region styleClass="icon, balance-icon"/>
         </graphic>
     </Label>
-    <Button fx:id="watchButton" mnemonicParsing="false"
-            onAction="#onWatchButtonClicked" GridPane.halignment="RIGHT" GridPane.valignment="BOTTOM"
-            GridPane.rowIndex="8" GridPane.columnIndex="2" GridPane.hgrow="ALWAYS" GridPane.vgrow="ALWAYS">
-        <graphic>
-            <Region styleClass="icon,play-icon"/>
-        </graphic>
-    </Button>
+
+
+    <HBox spacing="10.0" alignment="BOTTOM_RIGHT" GridPane.halignment="RIGHT" GridPane.valignment="BOTTOM"
+          GridPane.columnIndex="2" GridPane.rowIndex="8"
+          GridPane.hgrow="ALWAYS" GridPane.vgrow="ALWAYS">
+        <Button fx:id="deleteButton" mnemonicParsing="false"
+                visible="false" managed="false"
+                onAction="#onDeleteButtonClicked"
+                styleClass="delete-button">
+            <graphic>
+                <Region styleClass="icon,delete-icon"/>
+            </graphic>
```

Comment:
Updated this one to `uninstall-icon` like the maps and mods icons.

I think the buttons on the replay, maps, and mods section should match in style as much as possible, so I added color to those as well.

This may seem like a dramatic change, but I think colored buttons will highlight the shortcut to perform these actions directly from the card without the need to navigate into the detail view. Blue and Red make it clear what the buttons do without needing to recognize and process the meaning of the icon (less brain cells.) I think it will encourage players to actually use these buttons more.